VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -126,6 +127,7 @@
                         android:textColor="@color/white"
                         android:textSize="16sp"
                         android:padding="10dp"
+                        android:textAllCaps="true"
                         android:text="@string/view_button_text" />
```

Comment:
some android:text (like android:text="@string/view_button_text" )  is not changed to tools:text yet,    do we need to change them?